UnityProject/Assets/Scripts/Systems/FilthGenerator/FilthGenerator.cs

```diff
@@ -0,0 +1,93 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Chemistry;
+using Initialisation;
+using ScriptableObjects;
+using Shuttles;
+using TileManagement;
+using Tilemaps.Behaviours.Layers;
+using Tiles;
+using UnityEngine;
+using UnityEngine.Tilemaps;
+
+public class FilthGenerator : MonoBehaviour
```

Comment:
why not use a subsystem behavior like i've said because then you don't have to re-write duplicate code and let the systems handle running it when scenes finally load with the others